Framework/Algorithms/inc/MantidAlgorithms/SampleCorrections/IMCInteractionVolume.h

```diff
@@ -34,14 +34,12 @@ struct ComponentScatterPoint {
 class MANTID_ALGORITHMS_DLL IMCInteractionVolume {
 public:
   virtual ~IMCInteractionVolume() = default;
-  virtual bool calculateBeforeAfterTrack(
-      Kernel::PseudoRandomNumberGenerator &rng, const Kernel::V3D &startPos,
-      const Kernel::V3D &endPos, Geometry::Track &beforeScatter,
-      Geometry::Track &afterScatter, MCInteractionStatistics &stats) const = 0;
-  virtual double calculateAbsorption(const Geometry::Track &beforeScatter,
-                                     const Geometry::Track &afterScatter,
-                                     double lambdaBefore,
-                                     double lambdaAfter) const = 0;
+  virtual std::tuple<bool, std::shared_ptr<Geometry::Track>,
```

Comment:
Given this is used in a few locations would it be nice to define an alias to this, something like:

using ... = std::tuple<bool, std::shared_ptr<Geometry::Track>, std::shared_ptr<Geometry::Track>>